modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPSuccessTest.java

```diff
@@ -431,7 +451,13 @@ public void testAddIdP() throws IOException {
                 .log().ifValidationFails()
                 .assertThat()
                 .statusCode(HttpStatus.SC_CREATED)
-                .header(HttpHeaders.LOCATION, notNullValue());
+                .header(HttpHeaders.LOCATION, notNullValue())
+                .body("name", equalTo(IDP_NAME))
+                .body(baseIdentifier + "name", equalTo(new String(Base64.getDecoder().
+                        decode(SAMPLE_FEDERATED_AUTHENTICATOR_ID))))
+                .body(baseIdentifier + "isEnabled", equalTo(true))
+                .body(baseIdentifier + "definedBy", equalTo(DEFINED_BY_SYSTEM))
+                .body(baseIdentifier + "tags", Matchers.hasItems("Social-Login", "APIAuth"));
 
```

Comment:
Assert self.

Reply:
This is updated with e73225fc2e8068e574a4280d5206f44f5aaf2ceb